src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/Index.cshtml

```diff
@@ -0,0 +1,193 @@
+﻿@using SFA.DAS.Commitments.Support.SubSite.Extensions
+@using SFA.DAS.Commitments.Support.SubSite.Models
+@model ApprenticeshipViewModel
+@{
+    ViewBag.Title = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<div class="grid-row">
+
+    <div class="column-full">
+
+        <h1 class="heading-xlarge">
+            @Model.Name
+        </h1>
+
+        <div>
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Status</h2></th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                    <tr>
+                        <td>Agreement status</td>
+                        <td>@Model.AgreementStatus</td>
+
+                    </tr>
+                    <tr>
+                        <td>Payment status</td>
+                        <td>@Model.PaymentStatus</td>
+
+                    </tr>
+
+                    @if (Model.Alerts != null && Model.Alerts.Any())
+                    {
+                        for (int i = 0; i < Model.Alerts.Count(); i++)
+                        {
+                            if (i == 0)
+                            {
+                                <tr>
+                                    <td>Alerts</td>
+                                    <td><a href="#"> Model.Alerts[i]</a></td>
+
+                                </tr>
+                            }
+
+                            <tr>
+                                <td></td>
+                                <td><a href="#"> Model.Alerts[i]</a></td>
+
+                            </tr>
+                        }
+                    }
+                </tbody>
+            </table>
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Apprentice</h2></th>
+                    </tr>
+
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+                    <tr>
+                        <td>Name</td>
+                        <td>@Model.Name</td>
+
+                    </tr>
+                    <tr id="ULN">
+                        <td>Unique learner number</td>
+                        <td>@Model.ULN</td>
+                    </tr>
+                    <tr>
+                        <td>Date of birth</td>
+                        <td>@Model.DateOfBirth.ToGdsFormatWithSpaceSeperator()</td>
+                    </tr>
+                    <tr id="referenceNumber">
+                        <td>Cohort reference</td>
+                        <td>@Model.CohortReference</td>
+                    </tr>
+
+                    <tr id="referenceNumber">
```

Comment:
As mentioned below, we have duplicate (referenceNumber) ids here. If we don't reference the id's, we may as well remove them.